torch/fx/node.py

```diff
@@ -597,6 +598,7 @@ def format_node(
         self,
         placeholder_names: Optional[list[str]] = None,
         maybe_return_typename: Optional[list[str]] = None,
+        include_tensor_metadata: bool = False,
```

Comment:
nit: could you make this a kwarg?
```suggestion
        *,
        include_tensor_metadata: bool = False,
```